packages/odyssey-storybook/src/components/odyssey-mui/Accordion/Accordion.stories.tsx

```diff
@@ -123,3 +127,57 @@ export const Multi: StoryObj<AccordionProps> = {
     );
   },
 };
+
+export const Disabled: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+    isDisabled: true,
+  },
+  render: function C(props: AccordionProps) {
+    return (
+      <Accordion label="Title" isDisabled={props.isDisabled}>
+        {props.children}
+      </Accordion>
+    );
+  },
+};
+
+export const Expanded: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+  },
+  render: function C(props: AccordionProps) {
+    const [expanded, setExpanded] = useState(true);
+    const onChange = useCallback(
+      (_event: SyntheticEvent<Element>, expanded: boolean) =>
+        setExpanded(expanded),
+      [],
+    );
+    return (
+      <Accordion label="Title" isExpanded={expanded} onChange={onChange}>
+        {props.children}
+      </Accordion>
+    );
+  },
+  play: async ({ canvasElement, step }: PlaywrightProps<AccordionProps>) => {
+    await step("Accordion Expanded", async ({}) => {
+      const canvas = within(canvasElement);
+      const accordion = canvas.getByRole("button");
+      const accordionRegion = canvas.getByRole("region");
+
+      await waitFor(async () => {
+        await expect(accordionRegion).toBeVisible();
+      });
```

Comment:
Does `expect` return a promise here? The way the code's written, that's what it looks like, but I don't think it does. What we're wanting is to wait for it to not error within a timeframe.

In that case, we should just do this instead:
```suggestion
      await waitFor(() => {
        expect(accordionRegion).toBeVisible();
      });
```